### ISSUE TITLE
[Enhancement]: increase contrast between menus and app background

### ISSUE THREAD

**Issue by emilydrakesmith** [CrocSwap/ambient-ts-app#3164]
### Requirements

Several of our dropdown menus use the same background color as the general background of the app body (specifically `rgb(23, 29, 39) `). This causes accessibility issues because there isn't any contrast to distinguish the elements.

For both general usability and accommodation for visually-impaired users, no element with a given background color should be superimposed without interruption on top of another element with the same background color. When opening a dropdown, the user should be able to see before interacting exactly where are the bounds of that dropdown in the DOM without interacting.

We can rectify through one of several options:
1. **GOOD** Create a new background color which is only used as a background for dropdown menus with no other application in the codebase
2. **BETTER** Add a border around the dropdown menu, 2-3 px should be sufficient
3. **BEST** Both of the above options in concert

Network Switcher:
<img width="1840" alt="Explore ~ Ambient 2023-10-30 16-45-44" src="https://github.com/CrocSwap/ambient-ts-app/assets/71150790/4e61acbd-bf8a-4a92-bdd0-c495f535d1e8">

Resource Links:
<img width="1845" alt="Explore ~ Ambient 2023-10-30 16-50-09" src="https://github.com/CrocSwap/ambient-ts-app/assets/71150790/ff3ffe01-f002-44ae-a585-72948c17550f">

**Comment by miyuki-eto** [comment]
i have amended the header menu background colours and added a border. this is in the figma and can be found here - 

https://www.figma.com/file/0ItLk4nJmtj741R8sn3HYp/ambient?type=design&node-id=4438-61118&mode=design&t=GK2XcMpIXQwISxo7-4

<img width="1043" alt="image" src="https://github.com/CrocSwap/ambient-ts-app/assets/82160202/d4ca14cd-ecdd-45d1-9798-cea704c20ecc">
